text/0000-sensitive-data-redaction.md

```diff
@@ -0,0 +1,276 @@
+  - [Transform Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor)
+  - [Filter Processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor)
+
+### Alternative 0: Do nothing
+
+It’s always good to analyze this option. If we do nothing end users will still need to satisfy their requirements for treating sensitive data accordingly:
+
+- Instrumentation library authors are required to manage redaction before using the OpenTelemetry API
+- Application developers will do the same or are forced to join the instrumented application with a collector for redaction/filtering
+- Third party solutions will be implemented.
+
+### Alternative 1: OpenTelemetry Collector
+
+As listed above there are multiple processors available that can be used to redact or filter sensitive data with the OpenTelemetry collector. The challenge with that is that it is unknown to an application (owner) if data is processed in the collector as expected. Also, the data leaving the application might already be a risk (non-encrypted or compromised network) or may not be allowed (collector is hosted in a different country, which may conflict with a regulation)
+
+Ideally a combination is used.
+
+### Alternative 2: Backend
+
+The backend consuming the OpenTelemetry data can provide processing for filtering and redaction as well. The same objection as for the collector apply.
+
+### Existing Solutions outside OpenTelemetry
+
+There are many solutions outside OpenTelemetry that help to filter or redact sensitive data based on security and privacy requirements:
+
+- [sanitize_field_name in Elastic Java](https://www.elastic.co/guide/en/apm/agent/java/1.x/config-core.html#config-sanitize-field-names)
+- [Filter sensitive data in AppDynamics Java Agent](https://docs.appdynamics.com/appd/24.x/24.4/en/application-monitoring/install-app-server-agents/java-agent/administer-the-java-agent/filter-sensitive-data)
+- [GA4 data redaction](https://support.google.com/analytics/answer/13544947?sjid=3336918779004544977-EU)
+- [Configure Privacy Settings in Matamo](https://matomo.org/faq/general/configure-privacy-settings-in-matomo/)
+- [DataDog Sensitive Data Redaction](https://docs.datadoghq.com/observability_pipelines/sensitive_data_redaction/)
+
+## Open questions
+
+- **Question 1**: Should sensitivity details for an attribute in the semantic conventions be excluded from the stability guarantees? This means, updating them for a **stable** attribute is not a breaking change. The idea behind excluding them from the stability guarantees is that the requirements are subject of change due to changes in technology (see [#971](https://github.com/open-telemetry/semantic-conventions/pull/971), the list of query string values will evolve over time) or changes in regulatory requirements or both.
```

Comment:
I think yes.

Reply:
It feels like there should still be some stability definition.
I wouldn't want to see something that was redacted suddenly not be.

Reply:
That's a good point, maybe "adding redactions" may be outside, but "removing redactions" may require additional guard rails. It is probably unlikely that redactions will be removed, the only case I can think of is that something is deprecated for a very long time

Reply:
I will leave the question open, I can add some words on adding/removing redaction